test/distrib/bazel/test_single_bazel_version.sh

```diff
@@ -51,12 +51,16 @@ FAILED_TESTS=""
 export OVERRIDE_BAZEL_VERSION="$VERSION"
 # when running under bazel docker image, the workspace is read only.
 export OVERRIDE_BAZEL_WRAPPER_DOWNLOAD_DIR=/tmp
-bazel build -- //... "${EXCLUDED_TARGETS[@]}" || FAILED_TESTS="${FAILED_TESTS}Build "
+
+ACTION_ENV_FLAG="--action_env=bazel_cache_invalidate=version_${VERSION}"
+
+tools/bazel build "${ACTION_ENV_FLAG}" -- //... "${EXCLUDED_TARGETS[@]}" || FAILED_TESTS="${FAILED_TESTS}Build "
```

Comment:
qq: are the spaces in `FAILED_TESTS="${FAILED_TESTS}Build "`  and `FAILED_TESTS="${FAILED_TESTS}${TEST_DIRECTORY} Distribtest"` (line 63) where we want them? Seems that the pattern is different so I just wanted to doublecheck.

Reply:
You're right. Sloppy. Fixed.